firefox-ios/Client/Application/AccessibilityIdentifiers.swift

```diff
@@ -461,6 +461,8 @@ public struct AccessibilityIdentifiers {
 
         struct BookmarksPanel {
             static let tableView = "Bookmarks List"
+            static let title = "bookmarkTitle"
```

Comment:
I would name the property `titleTextFiled` and the content `BookmarkDetail.titleTextField` So we keep consistency with the other Accessibility identifiers.

Reply:
@FilippoZazzeroni  Updated in https://github.com/mozilla-mobile/firefox-ios/pull/22732/commits/4520906f768aaf063f992b8161f755de6cd73d26

Reply:
Thank you very much @hemche it looks great